### PR TITLE
Bug 1495829 - Stop Bintray. Update {group,artifact}Ids. Remove massage task

### PR DESCRIPTION
**Pull request by JohanLorenzo**

### CODE REVIEW

**Comment by pocmo** [comment]
happening

**Comment by JohanLorenzo** [comment]
Is this failure a known one @pocmo ? https://tools.taskcluster.net/groups/bDBfxvuqTLKyGa-5XBcLLg/tasks/RfCurwC7T26FOS7-C0x0DA/runs/0/logs/public%2Flogs%2Flive_backing.log#L812

**Comment by pocmo** [comment]
Yeah, that's #1083. It seems to happen quite often lately. :(

**Comment by JohanLorenzo** [comment]
Good to know! The PR graph isn't busted, and neither is the build task in the release graph: https://tools.taskcluster.net/groups/EIeM19SlRCW1SBHIoUQNAg/tasks/GA2hePj6TNeexKfLeceD9A/runs/0/artifacts. I ran CoT against some beetmover tasks and spot-checked the zips. I think we're good to go!

File: automation/taskcluster/decision_task_release.py [hunk 1]
```diff
@@ -50,8 +50,6 @@ def generate_build_task(version):
     checkout = ("git fetch origin --tags && "
                 "git config advice.detachedHead false && "
                 "git checkout {}".format(version))
-    bintray_publishing = (" && python automation/taskcluster/release/fetch-bintray-api-key.py"
-                          " && ./gradlew bintrayUpload --debug")
 
     assemble_task = 'assembleRelease'
     scopes = [
```
Comment (MihaiTabara) [inline automation/taskcluster/decision_task_release.py]: I think we can trash this scope as well from:

1. here 
2. from the [TC client.](https://tools.taskcluster.net/auth/roles/repo%3Agithub.com%2Fmozilla-mobile%2Fandroid-components%3Arelease)
3. from the main `.taskcluster.yml` file [here](https://github.com/mozilla-mobile/android-components/blob/master/.taskcluster.yml#L90)

**Comment by MihaiTabara** [comment]
Tracking bug is https://bugzilla.mozilla.org/show_bug.cgi?id=1495829.

File: README.md [hunk 2]
```diff
@@ -3,7 +3,6 @@
 [![Task Status](https://github.taskcluster.net/v1/repository/mozilla-mobile/android-components/master/badge.svg)](https://github.taskcluster.net/v1/repository/mozilla-mobile/android-components/master/latest)
 [![Build Status](https://travis-ci.org/mozilla-mobile/android-components.svg?branch=master)](https://travis-ci.org/mozilla-mobile/android-components)
 [![codecov](https://codecov.io/gh/mozilla-mobile/android-components/branch/master/graph/badge.svg)](https://codecov.io/gh/mozilla-mobile/android-components)
-![](https://api.bintray.com/packages/pocmo/Mozilla-Mobile/errorpages/images/download.svg)
```
Comment (pocmo) [inline README.md]: 🕵️

File: automation/taskcluster/decision_task_release.py (hunk 1 above)
Comment (JohanLorenzo) [reply automation/taskcluster/decision_task_release.py]: Great catch! I just removed it from 1 and 3. I'll remove from the TC client after we shipped the first release without bintray.

Comment (MihaiTabara) [reply automation/taskcluster/decision_task_release.py]: Makes perfect sense, ideal to have a fallback just in case things go wrong! 👍

**Pull request merged**